react-client/src/components/auth/resetPassword.component.js

```diff
@@ -5,8 +5,8 @@ class ResetPassword extends Component {
     handleSubmit = (event) => {
         event.preventDefault();
 
-        var form = new FormData(event.target);
-        var url = 'http://localhost:5000/auth/reset-password'
+        let form = new FormData(event.target);
+        let url = 'http://localhost:5000/auth/reset-password'
```

Comment:
Make 'http://localhost:5000' the base URL and then append other required parameters for the different URLs